services/web/server/src/simcore_service_webserver/projects/projects_api.py

```diff
@@ -199,7 +199,7 @@ async def _remove_services_and_data():
         )
         await delete_data_folders_of_project(app, project_uuid, user_id)
 
-    fire_and_forget_task(_remove_services_and_data())
+    await fire_and_forget_task(_remove_services_and_data())
```

Comment:
oh oh

Reply:
I wonder why this was not caught by the linter ...

Reply:
Wait a second, ``fire_and_forget_task`` is a **sync** (no A) function that creates a future and returns it.
No need for ``async``

Reply:
Apparently a [future can be awaited](https://docs.python.org/3/library/asyncio-future.html#asyncio.Future). But if I read it correctly this means, it awaits for the function to finish. And we have been using this `fire_and_forget_task` wrong.

Reply:
you do not wait for the fire_and_forget future
else it is of no use to call it that way

Reply:
removed await as it should. Now the tests properly catch the behaviour.